src/main/java/mate/academy/model/MovieSession.java

```diff
@@ -22,7 +22,7 @@ public class MovieSession {
     @ManyToOne(fetch = FetchType.LAZY)
     @JoinColumn(name = "cinema_hall_id")
     private CinemaHall cinemaHall;
-    @Column(name = "show_time")
+    @Column(name = "showTime")
```

Comment:
```suggestion
    @Column(name = "show_time")
```
Why did you change it?)